container-storage-setup.spec

```diff
@@ -1,12 +1,12 @@
-%define dsslibdir %{_prefix}/lib/docker-storage-setup
+%define dsslibdir %{_prefix}/lib/container-storage-setup
```

Comment:
Not in this patch set, next one.  You told me not to do dss->css, to keep the noise down.